tests_index.js

```diff
@@ -2,6 +2,7 @@ var testsContext = require.context('./test/specs', true, /spec\.js$/),
     srcContext = require.context('./src/charts', true, /\.js$/),
     jsFixturesContext = require.context('./test/fixtures', true, /\.js$/);
 
+
```

Comment:
Sure we need extra space?

Reply:
Nah, I was trying some stuff here that I finally didn't add. I will remove it.